hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/OzoneTestUtils.java

```diff
@@ -161,4 +161,37 @@ public static void closeContainer(StorageContainerManager scm,
             container.getState() == HddsProtos.LifeCycleState.CLOSED,
         200, 30000);
   }
+
+  /**
+   * Flush deleted block log & wait till something was flushed.
+   */
+  public static void waitForDeletedBlockLog(StorageContainerManager scm)
```

Comment:
Out of scope of this PR but I guess this test https://github.com/apache/ozone/blob/master/hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/hdds/scm/TestStorageContainerManager.java#L324-L334
can be rewritten using the new utility method here.

Can be done as part of [HDDS-11867](https://issues.apache.org/jira/browse/HDDS-11867)